composer.json

```diff
@@ -34,6 +34,7 @@
         "doctrine/orm": "^2.5",
         "doctrine/annotations": "^1.2",
         "friendsofsymfony/user-bundle": "^2.0@dev",
+        "guzzlehttp/guzzle": "^6.0",
```

Comment:
Maybe when it will be a PSR. But for now it has nos advantage over using Guzzle 6 for this use case: it is harder to setup (requires to install an extra Symfony bundle), adds complexity and a performance overhead.
Guzzle 6 is already an abstraction layer, with builtin curl and streams implementations. We only depend of the `ClientInterface`, it's easy to bridge other implementations (but I think it is not worth it).

As we support PHP 7 only, we don't target older libraries such as Guzzle 5.